src/List/MakeSelectable.js

```diff
@@ -87,6 +87,7 @@ export const MakeSelectable = (Component) => {
       const {
         children,
         selectedItemStyle,
+        ...componentProps,
```

Comment:
We use `other` everywhere else. Do you want to update your PR? Otherwise, I'm gonna carry your PR to master.